GUI/QtQuick/qml/WalletProperties/WalletPropertiesPopup.qml

```diff
@@ -32,7 +32,7 @@ CustomPopup {
 
         onSig_success: (nameExport, pathExport) => {
             _stack_view.push(success)
-            success.details_text = qsTr("Wallet %1 has successfully been exported to %2").arg(nameExport).arg(pathExport)
+            success.details_text = qsTr("Your watching-only wallet has successfully been exported\n\nFilename: %1\nFolder:      %2").arg(nameExport).arg(pathExport)
```

Comment:
Shouldn't we use a tab char (`\t`) for tabulation? Not sure if it's supported by the label, though. Indentation with space is a bad idea, IMO, as it may bork at font change.